widget/check_internal_test.go

```diff
@@ -270,19 +270,3 @@ func TestCheck_Disabled(t *testing.T) {
 	check.Enable()
 	assert.False(t, check.Disabled())
 }
-
-func TestCheckRenderer_ApplyTheme(t *testing.T) {
```

Comment:
Rather than removed shouldn't this be updated to test the alternative theme like we have done in other widgets (the "ugly" variant)?